stakepooldclient/stakepooldclient.go

```diff
@@ -704,3 +704,17 @@ func (s *StakepooldManager) GetStakeInfo() (*pb.GetStakeInfoResponse, error) {
 	}
 	return nil, errors.New("GetStakeInfo RPC failed on all stakepoold instances")
 }
+
+func (s *StakepooldManager) GetAllColdWalletExtPub() ([]*pb.GetColdWalletExtPubResponse, error) {
+	responses := make([]*pb.GetColdWalletExtPubResponse, len(s.grpcConnections))
+	for i, conn := range s.grpcConnections {
+		client := pb.NewStakepooldServiceClient(conn)
+		resp, err := client.GetColdWalletExtPub(context.Background(), &pb.GetColdWalletExtPubRequest{})
+		if err != nil {
+			log.Warnf("GetColdWalletExtPub RPC failed on stakepoold instance %s: %v", conn.Target(), err)
```

Comment:
I think this should be Errorf, as we cannot continue.  But really you can pass it on as this will log twice (three times actually).